src/sentry/event_manager.py

```diff
@@ -875,6 +883,19 @@ def save(self, project_id, raw=False, assume_normalized=False):
 
         return event
 
+    def _get_event(self, project_id, event_id):
+        if options.get("store.use-nodestore"):
```

Comment:
Could we have this flag set to a percentage? So we can try to roll it out gradually and assess the performance impact before we fully rely on nodestore? This would make the rollout safer.

Reply:
Yes, will update